configs/better-top.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+z=$(ps -aux | grep harmony)
```

Comment:
you have to filter out grep command itself, otherwise, you will always have two processes, one harmony, and one grep.

Reply:
It does return two entries. 

The cmd does a better job! 

$ps aux | egrep '[h]armony'